src/dialects/sqlite/query-interface.js

```diff
@@ -23,11 +23,41 @@ class SQLiteQueryInterface extends QueryInterface {
     const fields = await this.describeTable(tableName, options);
     delete fields[attributeName];
 
-    const sql = this.queryGenerator.removeColumnQuery(tableName, fields);
-    const subQueries = sql.split(';').filter(q => q !== '');
+    const savepointName = 'remove_column_savepoint';
```

Comment:
we should prefix our internal savepoints to be somewhat safe.
Or use a uuid, or a prefixed uuid. e.g. `sequelize_3958bbe4-284a-4d2f-9c5c-b136adc95774`

Reply:
Right, makes sense. 
Is there a helper function in the repo for generating uuids?

Reply:
the uuid package is available :)